base/applications/notepad/dialog.c

```diff
@@ -945,6 +944,7 @@ VOID DoShowHideStatusBar(VOID)
     ShowWindow(Globals.hStatusBar, (Globals.bShowStatusBar ? SW_SHOWNOACTIVATE : SW_HIDE));
 
     /* Update status bar contents */
+    DIALOG_StatusBarAlignParts();
     DIALOG_StatusBarUpdateAll();
```

Comment:
Suggestion: Add `Globals.bShowStatusBar` check as these updates are needed only if the status bar is visible.